doc/tools/make_rst.py

```diff
@@ -19,6 +19,28 @@
 MARKUP_ALLOWED_PRECEDENT = " -:/'\"<([{"
 MARKUP_ALLOWED_SUBSEQUENT = " -.,:;!?\\/'\")]}>"
 
+# Used to translate the section headings when required with --lang argument.
+# The HEADINGS list should be synced with what we actually write with `make_heading`,
+# and also hardcoded in `doc/translations/extract.py`.
+HEADINGS = [
```

Comment:
Note to self: When cherry-picking to `3.x`, the Constructors and Operators strings are not relevant.